contents/handbook/strategy/strategy.md

```diff
@@ -76,15 +76,28 @@ While there is value in the items on the right, we value the items on the left m
 
 * _Reject_: We enable our customers to ingest, store and analyze data on their infrastructure, we don't believe sending sensitive data to multiple cloud providers the right approach
 
-## Target audience for 2022
+## Direction for 2022
 
-Our customers are a combination of users, businesses and the products themselves. In order to understand where we should focus our efforts, below is a proposed breakdown of our target audience based on attributes of a product. 
+ * **2 word summary: **Nail Enterprise
+    * **Customers**
+        * **Focus on Enterprise.** E.g. Large initial contracts (~$200k/year+) and smaller deals in organizations that will eventually become very large. 
+        * **Non Goal:** Start doing outbound sales
+    * **Product**
+        * **Goals:**
+            * **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
+            * **Extensibility:** You can instantly connect PostHog to any other system 
+            * **Deployments:** PostHog is the easiest self-hosted product to deploy and scale in the world
+        * **Non Goals:**
+            * Build lots of new low quality / partial features
+            * Build everything a single enterprise customer wants just to close a deal
+
+## Target customers for 2022
+
+Our ideal customer is an Enterprise who meets as many of these these criteria as possible:
 
 * _Needs_
   * Need to control their user data
-  * Need to excel at product led 
 * _Haves_
   * Have budget and savvy engineers are the decision makers
-  * Have achieved product market fit
```

Comment:
@jamesefhawkins  I removed a couple since I'm not sure they're applicable if we filter on enterprise - would appreciate your thoughts?

Reply:
Removing product/market fit makes sense:

- the enterprise overall will have product market fit. Small teams (when we get adopted bottom up) may not BUT I thinkt eh value prop is so obvious to teams next door to them on other products that this isn't an issue.

I don't understand why we should remove need to excel at product led growth? Perhaps this should be clarified to "are building products primarily focussed on end users" versus ie some tool to lock ex employees out of their laptops where buyers and users are different groups.

Reply:
Sorry that was a typo on product led growth, I meant to add the word "growth" not delete the line.

Reply:
"are building products primarily focussed on end users" sounds fair, but maybe too broad, I think leaving as-is for now makes sense